### PR TITLE
SIMPLE:output proposal_interval in status for test debugging

### PR DESCRIPTION
**Pull request by jkrauska**
In testing, it's valuable to be able to query the daemon's proposal_interval. (eg. when making a txn, how long to wait for it to pass, or when checking for new blocks, what the expecting rate is..)

This simple PR adds the current proposal_interval to the status command.

### CODE REVIEW

File: src/lib/daemon_rpcs/daemon_rpcs.ml [hunk 1]
```diff
@@ -86,6 +87,7 @@ module Types = struct
             ("User_commands Sent", Int.to_string (f x)) :: acc )
           ~run_snark_worker:(fun acc x ->
             ("Snark Worker Running", Bool.to_string (f x)) :: acc )
+          ~proposal_interval:(fun acc x -> ("Proposal Interval", f x) :: acc)
```
Comment (nholland94) [inline src/lib/daemon_rpcs/daemon_rpcs.ml]: This should just pull the value from `Consensus.Mechanism.block_interval_ms` instead. "CODA_PROPOSAL_INTERVAL" is the config var specifically for proof of signature, and this PR also decouples the default parameter displayed from the default set in consensus.

Comment (jkrauska) [reply src/lib/daemon_rpcs/daemon_rpcs.ml]: @nholland94 cool - didn't know I could pull it from there.  will try.

Comment (jkrauska) [reply src/lib/daemon_rpcs/daemon_rpcs.ml]: @nholland94 using Consensus.mechanism

File: src/app/cli/src/coda_main.ml [hunk 2]
```diff
@@ -1092,8 +1092,7 @@ module Run (Config_in : Config_intf) (Program : Main_intf) = struct
     ; peers= List.map (peers t) ~f:(fun (p, _) -> Host_and_port.to_string p)
     ; user_commands_sent= !txn_count
     ; run_snark_worker= run_snark_worker t
-    ; proposal_interval=
-        Option.value (Unix.getenv "CODA_PROPOSAL_INTERVAL") ~default:"5000"
+    ; proposal_interval= Int64.to_int_exn Consensus.Mechanism.block_interval_ms
```
Comment (nholland94) [inline src/app/cli/src/coda_main.ml]: Really, you don't even need to store this in this record, you could just reference `Consensus.Mechanism.block_interval_ms` in `daemon_rpcs.ml`.

Comment (bkase) [reply src/app/cli/src/coda_main.ml]: it also needs to be in the record to come out in the JSON output

Comment (bkase) [reply src/app/cli/src/coda_main.ml]: for `status --json`

**Pull request merged**